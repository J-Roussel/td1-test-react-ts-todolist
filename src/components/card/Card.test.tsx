import renderer from "react-test-renderer";
import Card from "./Card";


describe('test Card componenet', () => {
    let id = 'test';
    let titre = 'test';
    let children = <div>test</div>
    it('renders the Card component', () => {
        const CARDCOMPONENT = renderer.create(<Card id={id} titre={titre} children={children}/>);
        let instance = CARDCOMPONENT.toJSON();
        expect(instance).toMatchSnapshot();
    })
})