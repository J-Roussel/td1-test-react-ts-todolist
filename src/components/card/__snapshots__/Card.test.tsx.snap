// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`test Card componenet renders the Card component 1`] = `
<div
  className="card-container"
>
  <h3
    className="title"
    id="test"
  >
    test
  </h3>
  <div>
    test
  </div>
  <p
    className="add-description"
  >
    Add 
    <span>
      +
    </span>
  </p>
</div>
`;
